controllers/provider-openstack/charts/images.yaml

```diff
@@ -13,4 +13,8 @@ images:
 - name: etcd-backup-restore
   sourceRepository: github.com/gardener/etcd-backup-restore
   repository: eu.gcr.io/gardener-project/gardener/etcdbrctl
-  tag: "0.6.4"
\ No newline at end of file
+  tag: "0.6.4"
+- name: openstack-cloud-controller-manager
+  sourceRepository: github.com/kubernetes/cloud-provider-openstack
+  repository: k8scloudprovider/openstack-cloud-controller-manager
+  tag: "v1.15.0"
```

Comment:
This would deploy the `openstack-cloud-controller-manager` in version `v1.15.0` for every shoot cluster (independent of the shoot version). This can't stay as it is.
I can see that there are releases of the out-of-tree CCM for OpenStack beginning with 1.11. You would probably need something like this:

```yaml
...
- name: openstack-cloud-controller-manager
  sourceRepository: github.com/kubernetes/cloud-provider-openstack
  repository: k8scloudprovider/openstack-cloud-controller-manager
  tag: v1.14.0
  targetVersion: 1.14.x
- name: openstack-cloud-controller-manager
  sourceRepository: github.com/kubernetes/cloud-provider-openstack
  repository: k8scloudprovider/openstack-cloud-controller-manager
  tag: v1.15.0
  targetVersion: 1.15.x
```

For Kubernetes version 1.11 I see the release is tagged [v0.2.0](https://github.com/kubernetes/cloud-provider-openstack/tree/v0.2.0). For 1.10 there is not even a release. Gardener is supporting shoot clusters  beginning with 1.10.

What's the plan now for 1.10 (and maybe also 1.11) clusters? Stick to the old in-tree CCM? In either case, the code needs to be adapted to reflect these scenarios.

Reply:
You have of course a point here. The in-tree support is the only way out here. Question is, do we have <1.11 clusters and what is the ETA when we will drop the support for that?

Reply:
We don't have ETA for that yet. For the time being we are still supporting 1.10, but there is already gardener/gardener#1128 which will remove support for it. However, it's not clear yet when we can/will merge it.